lib/commands/web.js

```diff
@@ -145,18 +231,66 @@ commands.setCookie = async function setCookie (cookie) {
   }
 
   const jsCookie = cookieUtils.createJSCookie(cookie.name, cookie.value, {
-    expires: _.isNumber(cookie.expiry) ? (new Date(cookie.expiry * 1000)).toUTCString() :
-      cookie.expiry,
+      expires: _.isNumber(cookie.expiry)
+        ? new Date(cookie.expiry * 1000).toUTCString()
+        : cookie.expiry,
     path: cookie.path,
     domain: cookie.domain,
     httpOnly: cookie.httpOnly,
-    secure: cookie.secure
+      secure: cookie.secure,
   });
   let script = `document.cookie = ${JSON.stringify(jsCookie)}`;
   await this.executeAtom('execute_script', [script, []]);
+  },
+  /**
+   * @this {XCUITestDriver}
+   */
+  async deleteCookie(cookieName) {
```

Comment:
Can we keep `lib/commands/cookies.js` for `getCookies`, `deleteCookie`, `deleteCookies` and `_deleteCookie` instead of here to not break the behavior?

So far, only `setCookie` is via `await this.executeAtom('execute_script', [script, []]);` but others are via web inspector implemented in `lib/commands/cookies.js` as xcuitest driver.

Reply:
AFAICT the code in `cookies.js` was never used and was overwritten by other mixins.  If you want to verify this yourself, take a look at `master`

Reply:
Hmm, okay. I'll test later. Btw, the current master has only setCookie
https://github.com/appium/appium-xcuitest-driver/blob/master/lib/commands/web.js#L135

Reply:
https://gist.github.com/KazuCocoa/b935fbca6efa1abf4688cc94bc14366a
Tested with Appium 1.22.3, XCUITest 4.19.1 and this branch

Reply:
I moved the stuff from `cookies` into web, e.g., `removeCookie`

Reply:
Missing commit?

In the removed cookies's code:
```
helpers._deleteCookie = async function _deleteCookie (cookie) {
  const url = `http${cookie.secure ? 's' : ''}://${cookie.domain}${cookie.path}`;
  return await this.remote.deleteCookie(cookie.name, url);
};
```

this current web.js's the same method's code:

```
  async _deleteCookie(cookieName) {
    this.log.debug(`Deleting cookie '${cookieName}'`);
    let webCookie = cookieUtils.expireCookie(cookieName, {path: '/'});
    let script = `document.cookie = ${JSON.stringify(webCookie)}`;
    await this.executeAtom('execute_script', [script, []]);
  },
```

Expected behavior to follow current master is `this.remote.deleteCookie(cookie.name, url);`'s code (from my previous comment's GIST)